.github/scripts/install-torch-tensorrt-windows.sh

```diff
@@ -0,0 +1,14 @@
+set -eou pipefail
+source "${BUILD_ENV_FILE}"
+
+# Install test index version of Torch and Torchvision
+${CONDA_RUN} ${PIP_INSTALL_TORCH} torchvision
+${CONDA_RUN} pip install pyyaml mpmath==1.3.0
+
+# Install TRT 10 from PyPi
+${CONDA_RUN} pip install tensorrt==10.0.0b6 tensorrt-${CU_VERSION::4}-bindings==10.0.0b6 tensorrt-${CU_VERSION::4}-libs==10.0.0b6 --extra-index-url https://pypi.nvidia.com
```

Comment:
Do we need both this file and the one in packaging? Also can we parse the dev_deps to get the TRT version?

Reply:
I think both are necessary in this case because one is for the `build-test-windows.yml` job and the other is for the `windows-test.yml` job; both files are referenced in a workflows.

Reply:
The parsing functionality has been added as well